### PR TITLE
LPS-159251 | Audit and refactor tests for SegmentationDisabled.testcase

### PR DESCRIPTION
**Pull request by rafaellajordao**
JIRA Ticket:
[LPS-159249](https://issues.liferay.com/browse/LPS-159249)
↳ [LPS-159251](https://issues.liferay.com/browse/LPS-159251)
Motivation:
Audit and refactor tests for SegmentationDisabled.testcase

### CODE REVIEW

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by rafaellajordao** [comment]
ci:test:tango

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationDisabledAtInstance.testcase [hunk 1]
```diff
@@ -31,87 +31,31 @@ definition {
...
-			Segmentation.disableConfigurationAtSystemSettings(checkLabel = "Enable Segmentation");
+			AssetListsAdmin.editAssetList(assetListTitle = "Dynamic Collection Test");
 		}
 
-		task ("Check alert when Segmentation Disabled") {
-			Segmentation.gotoSegmeentsService(key_portlet = "Instance Settings");
-
-			Segmentation.checkSegmentationDisabledWarning(textValue = "Segmentation is disabled in System Settings. To enable, go to System Settings.");
+		task ("Click at warning with link and confirm Instance Settings is open") {
+			Segmentation.clickToReEnableSegmentsWarningLink();
```
Comment (Luizcarlosqueiroz) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationDisabledAtInstance.testcase:53]: This macro has the same AssertVisible function is being used at the testcase, so it's a good idea to remove it from there (or just leave the click). There are some other parts with this situation.

**Review by Luizcarlosqueiroz — COMMENTED** [review]
Hey @rafaellajordao ! There some tests that have a macro doing the same assert from the subsequent function. I think the best idea is to remove the assert from the macros (or remove the hole macro is it's the case) :smile:

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationDisabledAtInstance.testcase (hunk 1 above)
Comment (rafaellajordao) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationDisabledAtInstance.testcase:53]: ok! good idea, i will fix!

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by rafaellajordao** [comment]
ci:test:tango

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by rafaellajordao** [comment]
ci:test:tango

**Review by Luizcarlosqueiroz — CHANGES_REQUESTED** [review]
Hey @rafaellajordao! Could you look at the obs?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationDisabledAtInstance.testcase [hunk 2]
```diff
@@ -31,87 +31,31 @@ definition {
 		}
 	}
 
-	@description = "LPS-152541 Given the 'Assign Roles by Segments' is disabled at System Settings
-    When user navigates to Segmentation at Instance Settings
-    Then should see an alert informing that is disabled and how to re-enable it "
-	@ignore = "true"
-	@priority = "4"
-	test CheckAssignRolesDisabledAlertAtInstanceSettings {
```
[the remark this answers is not in the document]
Comment (rafaellajordao) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationDisabledAtInstance.testcase:39]: I'll add, sorry!

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by rafaellajordao** [comment]
ci:test:tango

File: portal-web/test/functional/com/liferay/portalweb/macros/Segmentation.macro [hunk 3]
```diff
@@ -142,11 +124,6 @@ definition {
...
 	macro clickToReEnableSegmentsWarningLink {
 		Click(locator1 = "Segmentation#ENABLE_SEGMENTS_INSTANCE_SETTINGS");
-
-		AssertVisible(
-			key_configurationName = "Segments Service",
-			key_configurationScope = "Virtual Instance Scope",
-			locator1 = "SystemSettings#SCOPED_CONFIGURATION_NAME");
 	}
 
 	macro closeSegmentationDisabledWarning {
```
Comment (Luizcarlosqueiroz) [inline portal-web/test/functional/com/liferay/portalweb/macros/Segmentation.macro:129]: This macro is being used in some tests, and maybe there's no need for it. I created it, but it makes to assertions (one direct and another using a macro), when the goal should be only to close the alert. Do you think we should get rid of it?

Comment (Luizcarlosqueiroz) [reply portal-web/test/functional/com/liferay/portalweb/macros/Segmentation.macro:129]: Or maybe keep as a macro, but just with the click.

**Review by Luizcarlosqueiroz — COMMENTED** [review]
Hey @rafaellajordao! Sorry to suggest more changes, I noticed a macro that maybe can be removed.

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by rafaellajordao** [comment]
ci:test:tango

**Pull request closed without merging**